servicetalk-grpc-netty/src/main/java/io/servicetalk/grpc/netty/DefaultGrpcServerBuilder.java

```diff
@@ -197,6 +206,11 @@ protected void doAppendHttpServiceFilter(final Predicate<StreamingHttpRequest> p
 
     private HttpServerBuilder preBuild() {
         if (!invokedBuild) {
+            // TODO(dj): Consider logging a warning in every method that changes the state of the builder once it has
+            // been used to build a client.
```

Comment:
Don't understand this comment, can you clarify what you mean and why a client is involved here?